build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -2118,4 +2118,9 @@ message RequestMetadata {
   // There is no expectation that this value will have any particular structure,
   // or equality across invocations, though some client tools may offer these guarantees.
   string configuration_id = 7;
+
+  // If Bazel automatically retries an invocation, this value is set to the
```

Comment:
The API should be agnostic to the client, so it shouldn't call the client "Bazel".

Reply:
Oops, thanks for catching!